storage_service/locations/models/pipeline_local.py

```diff
@@ -36,6 +36,11 @@ class PipelineLocalFS(models.Model):
         help_text=_l("Name or IP of the remote machine."))
     # Space.path is the path on the remote machine
 
+    assume_rsync_daemon = models.BooleanField(default=False,
+        verbose_name="Assume remote host serving files with rsync daemon",
+        help_text="If checked, will use rsync daemon-style commands instead of the default rsync with remote shell transport")
+    rsync_password = models.CharField(max_length=64, blank=True, default="", help_text="RSYNC_PASSWORD value (rsync daemon)")
```

Comment:
The values of `verbose_name` and `help_text` should be translation strings, e.g.:

```python
verbose_name=_l("Assume remote host serving files with rsync daemon")  # This is 
```